Formula/bazel.rb

```diff
@@ -89,18 +89,5 @@ def install
 
     system bin/"bazel", "build", "//:bazel-test"
     assert_equal "Hi!\n", pipe_output("bazel-bin/bazel-test")
-
-    # Verify that `bazel` invokes Bazel's wrapper script, which delegates to
-    # project-specific `tools/bazel` if present. Invoking `bazel-VERSION`
-    # bypasses this behavior.
-    (testpath/"tools"/"bazel").write <<~EOS
-      #!/bin/bash
-      echo "stub-wrapper"
-      exit 1
-    EOS
-    (testpath/"tools/bazel").chmod 0755
-
-    assert_equal "stub-wrapper\n", shell_output("#{bin}/bazel --version", 1)
-    assert_equal "bazel #{version}-homebrew\n", shell_output("#{bin}/bazel-#{version} --version")
```

Comment:
I don't think we want to remove this test entirely. Just `return if OS.linux?` instead.

Reply:
yeah, let's see how it goes, and I can add `return if OS.linux? ` before this block.